src/game/default/g_types.h

```diff
@@ -482,8 +477,6 @@ typedef enum {
  * @brief Item types.
  */
 typedef enum {
-	ITEM_NONE,
-
```

Comment:
Gotcha. So here's my rationale:

g_item_t only exists, to both the people consuming them -and- developing them, as const members. The only instances of g_item_t are in g_itemlist, which is local to g_item.c and no instances ever escape that file, only *pointers* to instance. Unless a developer purposefully sets type to uninitialized (which in this case would be ammo, yeah), which would be really hard to do since they'd likely start by copypastaing other instances in the item list, there's absolutely no way that ITEM_NONE could ever be seen in regular gameplay *unless* there was a bug causing a g_item_t pointer to point outside of bounds, which again is.. extremely hard to do given the tools we have.

If you think we should have ITEM_NONE just for safety I can still do that, but there's like 8 other members there that will have to be filled with valid info before an item is "valid" anyways (name or classname for instance), so I don't think we're losing much by having an extra :)